src/layouts/CrossLayout.vue

```diff
@@ -201,7 +201,10 @@ export default {
   },
   mounted () {
     if (this.$q.lang.getLocale().indexOf('en') === 0) {
-      this.label = 'About Us'
+      import(`quasar/lang/${this.$q.lang.getLocale()}`).then(language => {
+        this.$q.lang.getLocale()(language.default)
+      })
+      this.label = this.$t('aboutus')
```

Comment:
we are getting closer to the point. I'll merge this. But I still not understand, why you are using a variable.